documentation/spherex_data_products.md

```diff
@@ -7,7 +7,11 @@ This summary includes filenaming conventions, for which we adopt the following d
 - `Planning Period` designates the survey plan uploaded to the spacecraft, e.g. `2025W18_2B`.
 Each planning period covers approximately 3.5 days of operation.
 
-- `Observation ID` includes the survey planning period and the large and small slew counters, e.g. `2025W18_2B_0001_1`.
+- `Observation ID` includes the survey planning period and the large
+  and small slew counters, e.g. `2025W18_2B_0001_1`. Each large slew
+  has a maximum of 4 small slews, so the allowed small slew counter
+  range is 1 to 4.  Some large slews will have less than 4 small
+  slews.
```

Comment:
A small thing, but I would recommend breaking lines at the end of sentences for narrative text rather than at a fixed line length